codingTest_jye/짝수와-홀수.js

```diff
@@ -0,0 +1,12 @@
+const num = 2;
+
+function solution(num) {
+    var answer = "Even";
+    if (num % 2){
```

Comment:
```suggestion
    if (num % 2 === 0){
```
```num % 2```는 결과값이 0 또는 0이 아닌 다른 값이 나옵니다. 문제 상황상 맞긴 하겠지만, 0, 1이 아닌 실제 값일 들어왔을 때, 홀수 짝수를 비교하는 로직에는 위에 로직이 더 적합한 것 같습니다.